### DISCUSSION TITLE
MultiSelect - allow wrapping of selected options

### DISCUSSION THREAD

**Discussion by omzy** [primefaces/community#866] [PrimeVue]
Following on from this closed issue: https://github.com/primefaces/primevue/issues/1730.

It would be good if there was an option to control whether selected options wrap on multiple lines instead of being cut off with ellipsis.

Currently, I am applying custom CSS to bring this behaviour, but this could cause problems in a future update.

**Comment by MiguelBamberino** [comment]
Good Morning,
Is there any update on this ticket ? What does "Resolution Waiting for 👍 " mean ?

thanks

**Comment by jamesslinger** [comment]
For anyone else wanting a workaround, you can use the following CSS;

.p-multiselect {
  min-width: 17rem;  // or any value
}
.p-multiselect-label {
  display: flex;
  flex-wrap: wrap;
}
.p-multiselect-token {
  margin: 0.1rem;  // this bit is optional 
}
